src/job-execution/WebJobExecutor.ts

```diff
@@ -142,12 +144,24 @@ export class WebJobExecutor extends BaseJobExecutor {
     }
 
     const requestPromise = new Promise((resolve, reject) => {
-      this.requestClient!.post(apiUrl, formData, undefined)
+      this.requestClient!.post(apiUrl, formData, authConfig?.access_token)
```

Comment:
again, I wonder if we need a comment to explain that the access_token is only relevant in SASjs (not Viya, for instance)

Reply:
added